hook.sh

```diff
@@ -9,7 +9,7 @@ function deploy_challenge {
     #   The domain name (CN or subject alternative name) being
     #   validated.
     # - TOKEN_FILENAME
-    #   The name of the file is irrelevant for the DNS challenge, yet still provided 
```

Comment:
I understand that spaces at the end of the line are crap, sorry for my poor code quality!

But can we please have a separate commit when removing those?